dbms/src/Interpreters/Settings.h

```diff
@@ -232,7 +232,10 @@ struct Settings
     M(SettingSeconds, max_execution_time, 0, "") \
     M(SettingOverflowMode<false>, timeout_overflow_mode, OverflowMode::THROW, "What to do when the limit is exceeded.") \
     \
-    M(SettingUInt64, min_execution_speed, 0, "In rows per second.") \
```

Comment:
maybe rename `min_execution_speed ` to `min_execution_speed_per_thread` is better ?  but this may cause version incompatibility.

Reply:
Better not to change.